run.sh

```diff
@@ -76,6 +79,9 @@ else
   printf "Already exists!\n"
 fi
 
+# Enable or Disable debug mode
```

Comment:
Not sure if I'm understanding.. Why is this needed if you're explicitly setting WP_debug with `wp-cli.yml`? If I'm reading this correctly, won't this just set the same value twice? Am I missing something?

If the goal is to generate a new `wp-config` file after build success (using a different option than was set initially), why not just write an npm run command to delete the current config file and then regenerate a new one using `wp core config [...]`?